DNN Platform/Library/Entities/Portals/PortalSettings.cs

```diff
@@ -132,6 +132,18 @@ public enum UserDeleteAction
             HardDelete = 3,
         }
 
+        public enum CspMode
+        {
+            /// <summary>Content Security Header is not added.</summary>
+            Off = 0,
+
+            /// <summary>Content Security Header is not added in Report Only.</summary>

```

Comment:
I believe this summary is incorrect and it should be "Content Security Header is added in Report Only mode."